### ISSUE TITLE
IllegalFormatConversionException on Long Search Queries in Message Search

### ISSUE THREAD

**Issue by monterinio** [obsidiandynamics/kafdrop#600]
### Description
When performing searches for messages in Kafdrop, if the input string (search text) takes a long time to execute, the application throws a java.util.IllegalFormatConversionException. This issue seems to occur due to a formatting error in KafkaMonitorImpl.java.

### Steps to Reproduce

1. Navigate to the message search functionality in Kafdrop.
2. Enter a search string that is expected to take a long time to execute.
3. Execute the search.

### Expected Behavior
The application should either return the search results or a message indicating that the search took too long, without any exceptions.

### Actual Behavior
The application throws a java.util.IllegalFormatConversionException. The stack trace is as follows:

```Stack trace:
java.util.IllegalFormatConversionException: d != java.util.Date
    at java.base/java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4442)
    at java.base/java.util.Formatter$FormatSpecifier.printInteger(Formatter.java:2963)
    at java.base/java.util.Formatter$FormatSpecifier.print(Formatter.java:2918)
    at java.base/java.util.Formatter.format(Formatter.java:2689)
    at java.base/java.util.Formatter.format(Formatter.java:2625)
    at java.base/java.lang.String.format(String.java:4145)
    at kafdrop.service.KafkaMonitorImpl.searchMessages(KafkaMonitorImpl.java:282)
    at kafdrop.controller.MessageController.searchMessageForm(MessageController.java:247)
    ...
```

### Possible Solution
It appears that the format specifier %d is incorrectly used for a java.util.Date object. This needs to be replaced with a proper date/time format specifier and formatted accordingly.

### Additional Information
Kafdrop version: 4.0.1

**Comment by monterinio** [comment]
Feel free to add me as a contributor so I can prepare a PR.

Update: 
https://github.com/obsidiandynamics/kafdrop/pull/601/files

**Comment by Bert-R** [comment]
Fixed through #601. Thanks!
